src/reasoning-layer-lib/connectors-reasoning-layer.xsl

```diff
@@ -386,37 +392,41 @@
         <xsl:variable name="connectorDirection" select="$connector/properties/@direction"/>
         <xsl:variable name="datatypeURI"
             select="f:buildURIfromLexicalQName('xsd:integer', fn:false())"/>
+<!--        this is first restriction content-->
```

Comment:
would be nice to find a way to refactor and optimise this in the future

Reply:
created a new ticket in the backlog